tasks/check_requirements.yml

```diff
@@ -3,7 +3,3 @@
   - name: Check if OS is Debian-based (we do not support others)
     debug: msg="Check OS family"
     failed_when: ansible_os_family != "Debian"
-
-  - name: Check if Ansible version is supported
-    debug: msg="Check Ansible version"
-    failed_when: ansible_version.full | version_compare('1.8', '<')
```

Comment:
That wouldn't be very useful: with Ansible 1.8, this check is never reached because the syntax is incompatible.